src/doc/swagger.json

```diff
@@ -219,7 +219,7 @@
         ]
       }
     },
-    "/products/nopromo": {
+    "/product/nopromo": {
```

Comment:
Pls leave this section this way with an `s`, I don't wanna allow the frontend to change the url and wait long time for build to get executed.

Reply:
done